recipes/config.rb

```diff
@@ -29,7 +29,7 @@ class ::Chef::Recipe
 
 if node['chef_client']['log_file'].is_a? String and node['chef_client']['init_style'] != 'runit'
   log_path = File.join(node['chef_client']['log_dir'], node['chef_client']['log_file'])
-  node.default['chef_client']['config']['log_location'] = "'#{log_path}'"
+  node.default['chef_client']['config']['log_location'] = "#{log_path}"
```

Comment:
Can't you just assign it to `log_path` directly? We don't need the string interpolation now.